### ISSUE TITLE
Any luck? Some notes on changes to the API, as well

### ISSUE THREAD

**Issue by joshuaswarren** [bassrock/homebridge-germguardian#1]
Just curious if you ever had any luck getting this to work, or any interest in continuing it. I've just gotten two of these GermGuadian air filters and as I started to sniff the traffic to document the API, my Google searching brought me to this repo. 

Also, in the couple of years since you created this repo, the URLs and process seems to have changed slightly. The current flow is:

POST to hub.mybeaconid.com/bcnapp/users/install
POST to hub.mybeaconid.com/bcnapp/users/registerDeviceToken
POST to hub.mybeaconid.com/bcnapp/users/registerDeviceToken (I'm wondering if this happens 2X because I have two air filters connected in the app)
POST to guardian.hubea.com/air/connect
POST to guardian.hubea.com/air/status
POST to guardian.hubea.com/air/command (I used the app to change the power level)
POST to guardian.hubea.com/air/disconnect

I've been able to make sense of all of these, except for the POST to registerDeviceToken, which I suspect is how we get the token needed for all of the guardian.hubea.com POSTs. The first POST to registerDeviceToken uses a content encoding of hce1, and the second POST uses a content encoding of hce2, and the request and response bodies are unintelligible. 

I did some digging, and mybeaconid.com appears to be a portal maintained by a company in Japan called Aplix that has a vague bit of documentation online https://business.aplix.co.jp/beacon/en/beacon_start.html - but it appears to all be related to Bluetooth beacons. I'm guessing the initial WiFi setup and configuration may be managed by a beacon-based process? Hubea.com is also owned by Aplix, but the entire site is in Japanese. It does seem to refer to beacons. 

Curious if anyone else has been able to make progress on this. At least from the searching I've done so far, this repo seems to represent the only info online about potentially connecting to and using the GermGuardian API.

**Comment by bassrock** [comment]
Hello! I’d love to continue this, but I just don’t have the time to continue trying to get the register device token to work.

If we can get past that it should be straightforward to get this to work.

**Comment by joshuaswarren** [comment]
Makes sense. I'm going a little stir crazy under a shelter in place order, so I'll keep digging into the registerDeviceToken flow and I'll followup here with anything I find.

**Comment by deanlyoung** [comment]
Is it possible they are using websockets? I believe Emerson Sensi thermostat uses websockets, which at least makes it difficult to re-implement if not impossible (but I don't know enough about websockets).

**Comment by SergeantPup** [comment]
I've been trying to do something with these air purifiers for years.  As mentioned above, this is STILL the most info out there.  I am going to take a crack at this because it looks like several of the pieces are here.  Would love to be able to API from Home Assistant rather than telling home assistant to tell Alexa to run a routine which calls the api.  I'll play with it this weekend and report back.  This will be my 16,473rd attempt at this.
